extensions/renderer/resources/app_window_custom_bindings.js

```diff
@@ -30,9 +30,10 @@ var try_hidden = function (view) {
 };
 
 var try_nw = function (view) {
-  if (view.nw)
-    return view;
-  return privates(view);
+  if (view.nw) {
+    return true;
+  }
+  return false;
 };
```

Comment:
This would break other places calling try_nw() since they expects objects, not boolean. Please keep this function unchanged.